tests/test_primary.py

```diff
@@ -149,7 +169,7 @@ def test_01_init(self):
           full_client_dir=TEMP_CLIENT_DIR,
           director_repo_name=demo.DIRECTOR_REPO_NAME,
           vin=5,  # INVALID
-          ecu_serial=primary_ecu_serial,
+          ecu_serial=PRIMARY_ECU_SERIAL,
```

Comment:
Is `ecu_serial` always the primary's ECU serial?  Maybe `primary_ecu_serial` would be a better name, similar to `primary_key`.  Alternatively, exclude the `primary` from `primary_key` so that the argument names are consistent.

Reply:
I think the latter makes the most sense. I'll do this in a separate PR, since it involves a lot of changes in primary.py and this is mostly a testing PR.